### PR TITLE
Fix lint errors in pulltorefresh.js

### PR DESCRIPTION
**Pull request by 3b3ziz**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Fixes some eslint errors:
1. Use the global form of 'use strict'
2. Unexpected unnamed function
## Related Tickets & Documents
https://github.com/thepracticaldev/dev.to/issues/3777
## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
NA
## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/assets/javascripts/lib/pulltorefresh.js [hunk 1]
```diff
@@ -54,319 +73,352 @@
 
   try {
     window.addEventListener('test', null, {
-      get passive() { // eslint-disable-line getter-return
-        _shared.supportsPassive = true;
+      get passive() {
+        // eslint-disable-line getter-return
```
Comment (rhymes) [inline app/assets/javascripts/lib/pulltorefresh.js]: this should be on the line of the `get passive`, or you can put it in the line before that and add `eslint-disable-next-line getter-return`

Comment (3b3ziz) [reply app/assets/javascripts/lib/pulltorefresh.js]: It was on the line of the function, however, lint-staged somehow changes this to be the line below it. I moved it to be above the line now. However, the error still shows.

**Comment by 3b3ziz** [comment]
Hello @rhymes, I am not sure how to test this. Would you please tell me if there are some known steps to test it?

However, I removed the WIP since I thought my changes are just variables names changing not breaking changes.

File: app/assets/javascripts/lib/pulltorefresh.js [hunk 2]
```diff
@@ -73,8 +73,8 @@
 
   try {
     window.addEventListener('test', null, {
+      // eslint-disable-line getter-return
```
Comment (rhymes) [inline app/assets/javascripts/lib/pulltorefresh.js]: `eslint-disable-next-line` :)

**Comment by rhymes** [comment]
Yeah, I was unsure how to test it too. I can't seem to trigger the "pull to refresh" with the simulate mobile modes in Firefox and Chrome. I used `brew install ngrok && ngrok http 3000` to have a website that I could point my smartphone to and it will redirect to my localhost. 

I still can't see the console to check if there are JS errors but the feature seems to be working.

Any ideas on this @jacobherrington ?

**Comment by 3b3ziz** [comment]
@rhymes I tested it on my mobile through localhost on the same network. It’s working fine.

**Pull request merged**